rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/websphere/ResolveWebSphereWsExtBindingXmlRuleProvider.java

```diff
@@ -1,48 +0,0 @@
-package org.jboss.windup.rules.apps.javaee.rules.websphere;
-
-import javax.enterprise.inject.Vetoed;
-
-import org.jboss.windup.config.GraphRewrite;
-import org.jboss.windup.config.metadata.RuleMetadata;
-import org.jboss.windup.config.phase.InitialAnalysisPhase;
-import org.jboss.windup.config.query.Query;
-import org.jboss.windup.config.ruleprovider.IteratingRuleProvider;
-import org.jboss.windup.graph.model.resource.FileModel;
-import org.jboss.windup.reporting.category.IssueCategoryRegistry;
-import org.jboss.windup.reporting.model.TechnologyTagLevel;
-import org.jboss.windup.reporting.service.ClassificationService;
-import org.jboss.windup.reporting.service.TechnologyTagService;
-import org.jboss.windup.rules.apps.xml.DiscoverXmlFilesRuleProvider;
-import org.jboss.windup.rules.apps.xml.model.XmlFileModel;
-import org.ocpsoft.rewrite.config.ConditionBuilder;
-import org.ocpsoft.rewrite.context.EvaluationContext;
-
-/**
- * Discovers WebSphere Web Service Binding XML files and parses the related metadata
- *
- * @author <a href="mailto:bradsdavis@gmail.com">Brad Davis</a>
- */
-// Disabled as a duplication of XML rule and with less information than what is on XML hint 
-@Vetoed
```

Comment:
I removed it because this rule searches for `ibm-webservices-bnd.xmi` configuration file but now this is done in rule [ResolveWebSphereWsBindingXmlRuleProvider](https://github.com/windup/windup/pull/1124/files/77d55c1c6b1a55c84fa3294392f8ef24d34afd7e#diff-570147a1ca26e7c81e74e75317718999R33) so it would be a "potential" duplicate. 
But if you feel it's better to keep the class with the @Vetoed annotation, i restore it.